spec/support/coronavirus_feature_steps.rb

```diff
@@ -68,8 +68,8 @@ def then_the_content_is_sent_to_publishing_api
     )
   end
 
-  def i_see_a_publish_landing_page_link
-    expect(page).to have_link(I18n.t("coronavirus.pages.index.landing_page_edit.something_else"))
+  def then_i_see_an_edit_landing_page_link
+    expect(page).to have_link(I18n.t("coronavirus.pages.index.landing_page_edit.sections"))
```

Comment:
I struggled with this for a bit - thinking why not just remove the test, but I can see why you've updated it to test the other link instead. Thanks for keeping a test on this page - totally the right thing to do 💯